index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <h1>Hello World, we are software developers!</h1>
+    <h1 id="title">Title</h1>
+    <p id="description">This is a survey form</p>
+    <form id="survey-form">
+      <label id="name-label" for="name">Name</label>
+      <input type="text" id="name" placeholder="Name" required />
+      <label id="email-label" for="email">Email</label>
+      <input type="email" id="email" placeholder="Email" required />
+      <label id="number-label" for="number">Number</label>
+      <input type="number" id="number" min="1" max="10" placeholder="Number" />
+      <select id="dropdown">
+        <option value="option 1">Option 1</option>
+        <option value="option 2">Option 2</option>
+      </select>
+      <label for="male">Male</label>
+      <input type="radio" name="gender" id="male" value="male" />
+      <label for="female">Female</label>
+      <input type="radio" name="gender" id="female" value="female" />
+      <input type="checkbox" name="name1" id="id-1" value="value-1" />
+      <input type="checkbox" name="name2" id="id-2" value="value-2" />
+      <input type="checkbox" name="name3" id="id-3" value="value-3" />
```

Comment:
I think it's would be be better to create more descriptive names, ids and values for these input elements.